meta.yaml

```diff
@@ -54,12 +55,14 @@ requirements:
     - matplotlib
     - nc-time-axis
     - netCDF4
-    - numpy
+    - numpy<=1.15.4
     - pandas
+    - palettable
     - python-cdo
     - python-stratify
     - pyyaml
     - scikit-learn
+    - seawater
```

Comment:
The seawater package seems to think of itself as outdated and only there for backwards compatibility, quote:

> The package uses the formulas from [...] (EOS-80).
> The EOS-80 library is considered now obsolete; it is provided here for compatibility with old scripts, and to allow a smooth transition to the new TEOS-10.

Could you please clarify and consider updating to the state-of-the-art anno 2010?

Reply:
Changed to gsw

Reply:
I actually changed it back to `seawater`. Since CMIP variables are still potential temperature and practical salinity we decide to stick to EOS-80. The `seawater` calls itself obsolete since it uses EOS-80 instead of TEOS-10, but EOS-80 is still widely used for the analysis and inside models as well.